mlflow/server/js/src/experiment-tracking/components/App.tsx

```diff
@@ -72,7 +72,7 @@ class App extends Component<AppProps> {
         {/* This layer enables intercompatibility between react-router APIs v5 and v6 */}
         {/* TODO: Remove after migrating to react-router v6 */}
         <CompatRouter>
-          <div style={{ height: '100%' }}>
+          <div css={{ display: 'flex', flexDirection: 'column', minHeight: '100vh' }}>
```

Comment:
Eh there's a problem with importing this line back to DB: many pages (particularly not existing in DB) would need to be `flex: 1`-ed as well 😕. I found general solution, PTAL at the PR comment